### PR TITLE
Hide status bar in participant list view correctly

### PR DESCRIPTION
**Pull request by ben181231**
connect SkygearIO/chat#77

### CODE REVIEW

File: SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift [hunk 1]
```diff
@@ -208,6 +213,7 @@ extension SKYChatParticipantListViewController: UITableViewDelegate, UITableView
         if let d = self.delegate {
             let participant = self.participants[indexPath.row]
             d.listViewController?(self, didSelectParticipant: participant)
+            self.dismiss(animated: true)
```
Comment (cheungpat) [inline SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift]: 1. I don’t know why this is suddenly here, the commit doesn’t seem related to view controller flow.
2. Suppose the user of this view controller class did not specify a delegate, you should still dismiss this view controller anyway (if dismissing is the intended behavior).

**Review by cheungpat — CHANGES_REQUESTED** [review]

File: SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift (hunk 1 above)
Comment (ben181231) [reply SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift]: Oh... That should not be here... Let me remove it.

**Comment by ben181231** [comment]
Updated.

**Review by cheungpat — APPROVED** [review]

**Comment by cheungpat** [comment]
Cherry-picked as bf3c762

**Pull request closed without merging**